pytorch_pfn_extras/nn/modules/extended_sequential.py

```diff
@@ -11,8 +12,18 @@ def _reset_parameters(model):
         for submodel in model.values():
             _reset_parameters(submodel)
     else:
-        if isinstance(model, torch.nn.Module):
+        if hasattr(model, 'reset_parameters'):
             model.reset_parameters()
+        elif hasattr(model, '_reset_parameters'):
+            model._reset_parameters()
+        else:
+            if len(list(model.parameters())) != 0:
```

Comment:
Maybe it's better to check buffers as well:
`len(list(model.parameters(recurse=False))) != 0 or len(list(model.buffers(recurse=False))) != 0`

Reply:
Thank you, fixed in 34eba76